coda-prototype/initUI.js

```diff
@@ -445,21 +445,19 @@ function initUI(dataset) {
 
     $("#export-dataset").click(() => FileUtils.saveDataset(newDataset));
 
-    $("#dataset-file").on("change", event => { // Fires when the dataset file has been changed by the file picker UI
+    /**
+     * Handles loading a new dataset after the user has selected a new dataset file via the Dataset dropdown.
+     */
+    $("#dataset-file").on("change", event => {
         $(event.target).parents(".dropdown").removeClass("open");
 
-        // Hide the existing alert.
-        let alert = $("#alert");
-        alert[0].childNodes.forEach(node => {
-            if (node.nodeName === "#text") {
-                node.remove();
-            }
-        });
-        alert.hide();
-        $(".tableFloatingHeaderOriginal").show();
+        UIUtils.hideAlert(); // This is a safety just in case there is a code path below which doesn't display an alert.
```

Comment:
Consider // ensure alert is hidden even if it isn't replaced

Reply:
changed.